tasks/build_control.js

```diff
@@ -160,6 +161,15 @@ module.exports = function (grunt) {
       }
     }
 
+
+    // Initialize the git config
+    function initConfig() {
+      for (var key in options.config) {
+        shelljs.exec('git config "' + key + '" ' + options.config[key]);
```

Comment:
If your name is `John Doe`, it only picks up `John`. Just need to add some qoutes to the option.config[key]

```
shelljs.exec('git config "' + key + '" ' + '"' + options.config[key] '"');
```